Tests/FluentAssertions.Equivalency.Specs/CollectionSpecs.cs

```diff
@@ -298,7 +298,7 @@ public void When_collection_of_same_count_does_not_match_it_should_include_at_mo
         const int commonLength = 11;
 
         // Subjects contains different values, because we want to distinguish them in the assertion message
-        var subject = new int[commonLength] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+        var subject = new[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
         var expectation = Enumerable.Repeat(20, commonLength).ToArray();
```

Comment:
I'm fine with simplifying the array initialization, but let's then also remove `commonLength` and use `Enumerable.Repeat(20, subject.Length)`

Reply:
Fixed in [899d468](https://github.com/fluentassertions/fluentassertions/pull/2470/commits/899d46880132f777a1a979dd4fa23a357b5a30ae).